### ISSUE TITLE
bokeh workers tab does not show worker name, just ips

### ISSUE THREAD

**Issue by alvarouc** [dask/distributed#1418]
Could you help me figure out how to show the worker names in the bokeh app?
I set the name as 
`dask-worker sch:8786 --name workername`

I would like to see the given name in the bokeh app.

**Comment by mrocklin** [comment]
Yes, currently the workers tab does not list this information.  There is a lot of information that we could list about workers.  It's not clear what is best to place on that page.
